### PR TITLE
(WIP) Update the profiler panel for Symfony 6.2+

### PR DESCRIPTION
**Pull request by javiereguiluz**
This is still WIP. When it's ready for review I'll post some screenshots.

### CODE REVIEW

File: Resources/views/Collector/db.html.twig [hunk 1]
```diff
@@ -98,289 +107,332 @@
...
-        <div class="metric">
-            <span class="value">{{ collector.querycount }}</span>
-            <span class="label">Database Queries</span>
-        </div>
-
-        <div class="metric">
-            <span class="value">{{ collector.groupedQueryCount }}</span>
-            <span class="label">Different statements</span>
-        </div>
+        <div class="metric-group">
```
Comment (stof) [inline Resources/views/Collector/db.html.twig:117]: Will this markup work in Symfony 6.1 too ?

File: Resources/views/Collector/db.html.twig [hunk 2]
```diff
@@ -98,289 +107,332 @@
...
+                <span class="label">Queries</span>
+            </div>
 
-        <div class="metric">
-            <span class="value">{{ '%0.2f'|format(collector.time * 1000) }} ms</span>
-            <span class="label">Query time</span>
-        </div>
+            <div class="metric">
+                <span class="value">{{ collector.groupedQueryCount }}</span>
+                <span class="label">Statements</span>
```
Comment (stof) [inline Resources/views/Collector/db.html.twig:125]: The previous label was `Different statements`, and the key thing to distinguish it from the previous number is the word `Different` that you removed.

File: Resources/views/Collector/db.html.twig [hunk 3]
```diff
@@ -98,289 +107,332 @@
...
-                    <span class="label">Misses</span>
-                </div>
+        </div>
 
-                <div class="metric">
-                    <span class="value">{{ collector.cacheCounts.puts }}</span>
-                    <span class="label">Puts</span>
-                </div>
+        <div class="tab">
+            <h3 class="tab-title">Connections & EM</h3>
```
Comment (stof) [inline Resources/views/Collector/db.html.twig:411]: should use `&amp;`

File: Resources/views/Collector/db.html.twig [hunk 4]
```diff
@@ -98,289 +107,332 @@
...
+                    <div class="empty">
+                        <p>Second Level Cache is not enabled.</p>
+                    </div>
+                {% else %}
+                    {% if not collector.cacheCounts %}
+                        <div class="empty">
+                            <p>Second level cache information is not available.</p>
+                        </div>
+                    {% else %}
+                        <div class="metric-gro">
```
Comment (HeahDude) [inline Resources/views/Collector/db.html.twig:374]:
```suggestion
                        <div class="metric-group">
```
?

**Comment by ostrolucky** [comment]
Sorry @javiereguiluz but this is open for too long with no traction. Let's reopen when you are ready to continue working on it, alright? :)

**Comment by nickygerritsen** [comment]
From the default profiler panes Doctrine is currently the only one that is not in the new style, which is most obvious looking at the sidebar:
image

What needs to happen to actually make this update? Maybe we can resurrect/recreate the PR? Or should we at least create an issue to track that this is something that needs to be done?

**Pull request closed without merging**